docs/development.md

```diff
@@ -0,0 +1,60 @@
+Development environment
+======================
+
+The purpose of the development environment is only for running the different test and metric tools.
+
+To get started, first setup the development environment. The dev. env. is a virtual machine. Every task described is run
+from that machine.  
+
+Requirements
+-------------------
+- Vagrant 2.2.x
+    - vagrant-hostsupdater (1.1.1.160, global, optional)
+    - vagrant-vbguest (0.19.0, global)
+- Virtualbox
+
+Install
+-------------------
+
+### 1. Create virtual machine
+
+``` cd homestead ``` 
+ 
+``` composer install ```
```

Comment:
If you're doing this then Composer would be a requirement.

Reply:
👍